### DISCUSSION TITLE
what version of C++ we can use for openframeworks development. C++23 vs C++17

### DISCUSSION THREAD

**Discussion by SemanticZeus** [openframeworks/openFrameworks#8221] [General]
Hi.
right now I'm working on improving the compatibility of the ofxSvg addon.
I wrote a few functions that alternate the input SVG file in memory so that ofxSvg can process them.
I'm going to send a pull request shortly.
but my question is what is the maximum version of C++ we are allowed for the internal development of openframewroks.
I'm using C++23, but if we are limited to C++17 I should modify my code before sending a pull request.

thank you.

**Comment by NickHardeman** [comment]
You might want to target C++17 for compatibility and more legacy support.
